horace_core/sqw/@Experiment/Experiment.m

```diff
@@ -259,6 +259,13 @@
         function obj = set.runid_recalculated(obj,val)
             % Do not normally use, except for tests. This is internal
             % property, which inform about the behaviour of runid map
+            % Inputs:
+            % val  -- either containers.Map which define correspondence
+            %         between run_id and header index or array of unique 
+            %         numbers which will become new run_id. 
+            %
+            %         Syncrhonization with run_id stored in pixels have to
+            %         be performed separately.
```

Comment:
```suggestion
            % val  -- either:
            %         - containers.Map which defines correspondence
            %         between run_id and header index 
            %         OR 
            %         - array of unique numbers which will become new run_id. 
            %
            %         Synchronization with run_id stored in pixels has to
            %         be performed separately.
```

Reply:
Thsis is absolutely wrong and confusing comment. Removing it.